docs/developers-guide.md

```diff
@@ -121,6 +121,25 @@ the ``klee_add_component`` function.
 
 That's it! 
 
+### Tracing
+
+KLEE provides macros that aid with tracing or "print debugging".
+
+`KLEE_DEBUG_WITH_TYPE(type, code);` which can be used as follows:
+
+```C
+#include "klee/Support/Debug.h"
+...
+KLEE_DEBUG_WITH_TYPE("my-feature",
+  llvm::errs() << "Currently there are " << states.size() << "\n";
```

Comment:
Maybe use `dbgs` and add " states"?
`llvm::dbgs() << "Currently there are " << states.size() << " states\n"`